src/main/scala/li/cil/oc/Settings.scala

```diff
@@ -523,6 +542,12 @@ object Settings {
     // Upgrading to version 1.5.20, changed relay delay default.
     VersionRange.createFromVersionSpec("[0.0, 1.5.20)") -> Array(
       "switch.relayDelayUpgrade"
+    ),
+    // Upgrading past version 1.7.0 (?), changed wireless card stuff for t1 card.
+    // TODO Update next version when its known what it will actually be
+    VersionRange.createFromVersionSpec("[0.0, 1.7.1)") -> Array(
+      "power.cost.wirelessCostPerRange",
+      "misc.maxWirelessRange"
```

Comment:
and ports?

Reply:
tbh, I don't know how this is used

Reply:
The old port setting does not need to be changed because 16 for wired cards is find, wireless is a new setting.